### ISSUE TITLE
Central Initialisation of Variables

### ISSUE THREAD

**Issue by Aarush-Acharya** [AOSSIE-Org/Resonate#357]
### 🚀 The feature

Currently many redundant variables are being initialised in different files. A system should be established which initialises variables only once and other files just use the reference of that variables initialised in that file

Take example of the Explore Controller written and initialised only once globally in the explore_controller.dart and the profile_screen.dart and explore_screen.dart just use that reference after importing explore_controller.dart in them

Get rid of all redundant initialisations and achieve the as explained desired state

### Motivation, pitch

Having less to no redundant initialisations would avoid the extra use of memory required why running the app thus boosting its performance

**Comment by M4dhav** [comment]
Hey, I can work on this. I have previously worked on similar stuff, please assign

**Comment by Aarush-Acharya** [comment]
Hey @M4dhav, reduce as much redundancy in initializations as you can. Make sure to write clean and organised thinking at each step what code be made better to the system you design to add a better value to the project.

**Comment by Aarush-Acharya** [comment]
Hey I have merged a PR please pull before working further and make sure to work on the dev branch

**Comment by Aarush-Acharya** [comment]
Hey new PR merged please pull before moving forward

**Comment by vanshambansal** [comment]
hey!!! is the isssue open ?

**Comment by M4dhav** [comment]
@vanshambansal I am working on the issue

**Comment by dikshantbhangala** [comment]
@Aarush-Acharya can you assign this issue to me???

**Comment by rohhann12** [comment]
@M4dhav are you on it?

**Comment by M4dhav** [comment]
@rohhann12 yes, I am on it

**Comment by rohhann12** [comment]
@M4dhav I just think you are focusing on a lot of issues, do you mind if I take on this one?

**Comment by M4dhav** [comment]
@rohhann12 I'm currently working on this and #359 only and since both require extensively going through the codebase, the workflow for both overlaps and is similar. Moreover I've already done work on both of the issues. So i apologise but i cannot give you this issue.

**Comment by rohhann12** [comment]
No problem,all the best 🙂

**Comment by Aarush-Acharya** [comment]
Hey @M4dhav, any update on this once done please raise the PR

**Comment by M4dhav** [comment]
@Aarush-Acharya will raise PR soon
